drgn/helpers/linux/printk.py

```diff
@@ -243,7 +246,7 @@ def get_printk_records(prog: Program) -> List[PrintkRecord]:
 
 
 @takes_program_or_default
-def get_dmesg(prog: Program) -> bytes:
+def get_dmesg(prog: Program, human_readable_time: bool = False) -> bytes:
```

Comment:
I'd prefer making this parameter keyword-only:
```suggestion
def get_dmesg(prog: Program, *, human_readable_time: bool = False) -> bytes:
```
It should also be documented (i.e., `:param human_readable_time: description...` in the docstring).